src/components/shared/Copyable.tsx

```diff
@@ -14,17 +15,16 @@ const TooltipWrapper = styled.div`
 `;
 
 const Wrapper = styled.div`
-  .copyable {
```

Comment:
This broke at some point during the SASS migration. The copy icon was no longer displayed